examples/pymc3_howto/sampling_compound_step.ipynb

```diff
@@ -12,10 +12,24 @@
     "# Compound Steps in Sampling\n",
```

Comment:
<p>I would not list the available step methods here as it will get outdated in no time. I would also move that to the compound steps by default secion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>